Gemfile

```diff
@@ -1,9 +1,10 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
-ruby '3.0.2'
+ruby '3.1.2'
 
 gem 'jekyll'
 gem 'puma'
+gem 'rack', '< 3'
```

Comment:
Why that ?

Reply:
because rack 3 is not supported yet by puma